### DISCUSSION TITLE
增减自定义主题的接口，并且能自定义字体

### DISCUSSION THREAD

**Discussion by xingnengmao666** [Meloong-Git/PCL#4399] [讨论]
增减自定义主题的接口，并且能自定义字体

**Comment by hejiehao** [comment]
> 能自定义字体

https://github.com/Hex-Dragon/PCL2/issues/366

> 增减自定义主题的接口

有点像 https://github.com/Hex-Dragon/PCL2/issues/3202

**Comment by allMagicNB** [comment]
你要什么接口？

**Comment by Ad-closeNN** [comment]
> 能自定义字体

可以是可以...不过要用到开源版本（而且修改后可能有其他问题），把开源仓库里的font.ttf改为你的font（要把那个字体改名为`PCL English`，字体名，不是文件名）然后编译就行了
~~应该不会有人为了自定义字体而用开源版本吧？~~
